### DISCUSSION TITLE
graphscope支持链接数据库吗？

### DISCUSSION THREAD

**Discussion by wulizzh** [alibaba/GraphScope#3443] [General]
graphscope支持链接数据库吗？

**Comment by yecol** [comment]
Hi Wulizzh, 

Currently GraphScope does not support direct link to databases.
However, we are work on a project named [GART](https://github.com/GraphScope/gart), to support synchronizing data from relational databases. Please check out the repo to see whether it fits your needs. 

/cc @doudoubobo @ds-ssj
